### DISCUSSION TITLE
Question regarding combined estimations

### DISCUSSION THREAD

**Discussion by AnasAlzahrani** [moj-analytical-services/splink#184] [General]
In the demo for combined estimation, Why do you append all columns into the setting ? 

line 15
```
settings_combined["blocking_rules"].append('l.email=r.email')
settings_combined["blocking_rules"].append('l.city=r.city')
```

**Comment by RobinL** [comment]
The idea is when this  trained model is then used,  these additional blocking rules are used to generate record comparisons

i.e. record comparisons are generated according to either:
email matches
or
city matches
or
(any existing blocking rules)

And then all of these record comparisons are scored according to the match weights in the combined trained model
